shared/utils/tool-homepages.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const browserify = 'http://browserify.org/';
+export const rollup = 'https://rollupjs.org/';
+export const webpack = 'https://webpack.js.org/';
+export const parcel = 'https://parceljs.org/';
```

Comment:
We should probably integrate this into `config.js`. 

I’d say let’s open an issue and do that in another PR as that one will have wide-reaching changes.

Reply:
Maybe link to v2.parceljs.org rather than the v1 site, since that's what's tested on tooling.report?

Reply:
Good shout!

Reply:
```suggestion
export const parcel = 'https://v2.parceljs.org/';
```

Reply:
Filed #383